packages/venia-concept/webpack.config.js

```diff
@@ -63,7 +74,10 @@ module.exports = async env => {
              * the globals object in jest.config.js.
              */
             POSSIBLE_TYPES: JSON.stringify(possibleTypes),
-            STORE_NAME: JSON.stringify('Venia'),
+            STORE_NAME:
+                process.env.STORE_VIEW_CODE && availableStore
+                    ? JSON.stringify(availableStore.store_name)
+                    : JSON.stringify(storeConfigData.store_name),
```

Comment:
This can also be simplified. If we have a matching `availableStore` we should just use that value, otherwise fall back to the default store name.

```suggestion
            STORE_NAME: availableStore
                    ? JSON.stringify(availableStore.store_name)
                    : JSON.stringify(storeConfigData.store_name),
```